cmd/zrepl/zrepl.c

```diff
@@ -186,25 +187,24 @@ zio_cmd_free(zvol_io_cmd_t **cmd)
 
 
 static int
-uzfs_zvol_socket_read(int fd, char *buf, uint64_t nbytes)
+uzfs_zvol_socket_read(int fd, char *buf, uint64_t nbytes, int flags)
```

Comment:
does uzfs_zvol_socket_write also need to be extended similar to this?

Reply:
OK.